lib/timeval.c

```diff
@@ -48,7 +48,7 @@ struct timeval curlx_tvnow(void)
 
```

Comment:
I agree this is the right move. What do you think about a name like curltime instead of curlval to know it's time related? The (long)s above need to be changed to time_t and unsigned, otherwise looks fine